pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.output

```diff
@@ -6,7 +6,7 @@ All control plane nodes successfully updated to 4.16.0-ec.3
 = Worker Upgrade =
 
 WORKER POOL   ASSESSMENT   COMPLETION    STATUS
-worker        Degraded     37% (22/59)   44 Available, 5 Progressing, 12 Draining, 7 Degraded
+worker        Degraded     37% (22/59)   44 Available (23 Outdated), 15 Unavailable (Progressing 5), 5 Progressing, 12 Draining, 7 Degraded
```

Comment:
I am wondering whether in the `X Unavailable (Progressing Y), Z Progressing` output the `Y` and `Z` values will always be equal and thus will render one of the references redundant. I would suppose a progressing node is also always unavailable, is it not?

Reply:
Unavailable is from MCO.
A node is unavailable if
- it has problems that need admins' attention. We had a card for the reason etc. My feeling is that an unavailable node in this case could be NOT progressing.
- MCO is updating the node. An unavailable node in this case should be progressing as well.

Reply:
hmm.. definitely a lot of dilemmas on how to UX this right. moreover its pretty visible so "better be done right" :wink: 
honestly, `X Unavailable (Progressing Y), Z Progressing` looks a bit strange to my eyes.
ideally, those should be made as obvious for the many as possible (in which case either displaying too much or too little is an enemy). that unfortunately is more of a finding a middle-ground between different personal taste, than solving an actual technical challenge.

Reply:
tbh unavailable itself sounds a bit ambiguous, specifically because it can have either "the good" or "the bad" kind of unavailable. I wish my linguistical skill could provide a simple yet accurate word to name those two cases distinguishably..

Reply:
**Brainstorming outputs. I am not suggesting or recommending any of the outputs.**
```
worker        Degraded     37% (22/59)   44 Available (23 Outdated), 15 Unavailable (5 Progressing), 12 Draining, 7 Degraded
```
or 
```
worker        Degraded     37% (22/59)   44 Available (23 Outdated), 15 Unavailable, 5 Progressing, 12 Draining, 7 Degraded
```

> tbh unavailable itself sounds a bit ambiguous, specifically because it can have either "the good" or "the bad" kind of unavailable.

Yes. That is the goal. Not that simple.

One extreme is having something like `"15 are unavailable (5 of them are due to the update; this is normal; the other ones can be problematic)"`. The other one is having users take the total from the `completion` column, subtract `available`, and assuming users know that a progressing node is unavailable, and subtract that from the unavailable. A little bit complex. 

For example, my first output tries to tie the progressing to the unavailable (there is a connection between them; 15 are unavailable, but 5 of them are progressing (but maybe it's not that clear?) - the current PR change), and removes the second reference of the progressing (as it is probably redundant IMO). 

```
worker        Degraded     37% (22/59)   44 Available (23 Outdated), 15 Unavailable (5 Progressing), 12 Draining, 7 Degraded
```

Reply:
The PR does more things at once:

- The `44 Available (23 Outdated)`: I do not see the value of this (which obv does not mean there isnt any, but I'll need someone to convince me) 
- The `15 Unavailable (5 Progressing)`: This is useful, but just partially correct. Even other "updting" states, such as "draining" cause unavailability. I'd like to get to the point where we somehow explain that some unavailable  nodes are good, some are not (maybe we stop using "unavailable" entirely? I don't know) but that seems like a larger problem than "line simplification". It is also closely related to unavailability "budgets" - that's why I at first thought this PR is trying to tackle that.